### PR TITLE
Add Frege

### PR DESCRIPTION
**Pull request by sfischer13**
**https://github.com/sfischer13/awesome-frege**

**This list is about Frege, a Haskell for the JVM. It contains awesome resources for Frege developers.**

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Feedback:

- Drop the `Inspired by the awesome list thing.` text.
- Drop the Travis status badge. It has no practical purpose there.
- The contributing guidelines should also include the expected formatting. See contributing.md in this repo.
- Include project logo in the top right, like e.g. https://github.com/sindresorhus/awesome-electron (The logo should link to the project website) (The logo should be high-DPI, set it to minimum half the width of the original image)
- Drop all the `A` / `An` prefixes in the descriptions.
- In the description at the top, `A curated list of awesome resources for the Frege programming language.`, linkify `Frege` to https://en.wikipedia.org/wiki/Frege_(programming_language)

File: readme.md [hunk 1]
```diff
@@ -127,6 +127,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [AutoIt](https://github.com/J2TeaM/awesome-AutoIt)
 - [Crystal](https://github.com/veelenga/awesome-crystal)
 - [TypeScript](https://github.com/dzharii/awesome-typescript)
+- [Frege](https://github.com/sfischer13/awesome-frege)
```
Comment (sindresorhus) [inline readme.md]: Add a succinct description. See other entries with description on how to format it.

Comment (sfischer13) [inline readme.md]: Thanks for the detailed feedback! I'm working on the changes.

**Comment by sfischer13** [comment]
- [x] Drop the Inspired by the awesome list thing. text.
- [x] Drop the Travis status badge. It has no practical purpose there.
- [x] The contributing guidelines should also include the expected formatting.
- [x] Include project logo in the top right.
- [x] Drop all the A / An prefixes in the descriptions.
- [x] In the description at the top, linkify Frege.

- [ ] Add a succinct description.

**Comment by sfischer13** [comment]
OK, everything should be fixed now.

**Comment by sindresorhus** [comment]
👌✨

**Comment by sfischer13** [comment]
Thanks again for your effort and for creating this cool project!

**Pull request merged**